test_data/encoding.csv

```diff
@@ -127,3 +127,16 @@
 68.3500147997595,113.625636875353,15,9PWM9J2G+272FWJV
 38.1176000887231,165.441989844555,15,8VC74C9R+2QX445C
 -28.1217794010122,-154.066811473758,15,5337VWHM+77PR2GR
+#
+# Test for poor performance, which may be a security issue (denial of service)
+# see https://github.com/google/open-location-code/issues/373
+# and https://github.com/google/open-location-code/pull/370
+-80.000000,-170.000000,2,22000000+
+-80.000000,-36170.000000,2,22000000+
```

Comment:
NIT: Would the first and last case suffice?

Reply:
Yes. Updated at a89ce4cbf729348c06842bf81bb3eb9c80fa1df3